src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -787,6 +796,11 @@ def main(argv):
     arguments.append(constants.VALUE_PROFILE_ARGUMENT)
     fuzzing_strategies.append(strategy.VALUE_PROFILE_STRATEGY)
 
+  if do_fork():
+    # TODO(metzman): Use `-fork=2` on Windows.
+    arguments.append('%s%d' % (constants.FORK_FLAG, 1))
```

Comment:
What does the value of this argument mean? Would it be better to move it into another constant and use on both lines 801 and 802. Would be a bit more convenient when you enable `2`.

Reply:
Done